src/main.ts

```diff
@@ -47,20 +43,28 @@ export default async function run() {
     }
 }
 
+function setUserAgentVariable(): void {
```

Comment:
Is this UA construction defined/specced anywhere that you could add a link to?

Reply:
I'm not sure, couldn't find any documentation. This logic was already there, I just moved it to its own private function. I'll keep an eye out in the docs if I come across anything like this.